File: Coursework.API/Core/Entities/Origin/Player.cs
using Core.Models;
using System;
using System.Collections.Generic;
using System.Text;

namespace Core.Entities.Origin
{
    class Player : IEntity
    {
        public string Id { get; set; }


        public string Nickname { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime DateOfBirth { get; set; }

        public string TeamId { get; set; }
        public Team Team { get; set; }

        public IEnumerator<BioMeasure> BioMeasures { get; set; }
    }
}

File: coursework-front-end/src/app/models/player.ts
import { IModel } from './imodel';

export class Player implements IModel{
    constructor(
        public id?: string,
        public nickname?: string, 
        public firstName?: string,
        public lasttName?: string,
        public dateOfBirth?: Date,
        public teamId?: string,
    ) { }

    public static mock(): Player {
        return new Player("Mock", "Mocky", "Mo", "Ck", new Date(), "Some Team Id");
    }
}